addon/components/sl-date-time.js

```diff
@@ -169,8 +185,15 @@ export default Ember.Component.extend( TooltipEnabled, {
         'timezone',
         'momentValue',
         function() {
-            return this.get( 'momentValue' )
-                .tz( this.get( 'timezone' ) ).format( 'z' );
+            const momentValue = this.get( 'momentValue' ),
+                timezone = this.get( 'timezone' );
+
+            if ( momentValue && timezone ) {
+                return momentValue
```

Comment:
Either place all of these calls on the same line (preferred) or put each new method call on its own line.